shared/common-adapters/follow-button.desktop.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import React, {Component} from 'react'
+import {Button} from './'
+import type {Props} from './follow-button'
+
+type State = {mouseOver: boolean}
+
+export default class FollowButton extends Component<void, Props, State> {
```

Comment:
This component implements the hover "Following" -> "Unfollow" behavior on desktop.